src/api/app/models/decision_favored_with_comment_moderation.rb

```diff
@@ -7,7 +7,7 @@ def description
   end
 
   def self.display_name
-    'favored with comment moderation'
+    'favor with comment moderation'
```

Comment:
A slight nitpick, but it might be nicer to say "favor and moderate the comment" instead, since it indicates what is going to happen a bit better imo

Reply:
pushed